lib/auth_web/controllers/permission_controller.ex

```diff
@@ -0,0 +1,61 @@
+defmodule AuthWeb.PermissionController do
```

Comment:
Does there need to be some authentication so only certain people can make permissions?

Reply:
Right now permissions don't do anything. But yes, these will be restricted shortly. 👍